easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.19-intel-2016a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'Ghostscript'
+version = '9.19'
+
+homepage = 'http://ghostscript.com'
+description = """Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
+ different targets. It used to be part of the cups printing stack, but is no longer used for that."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'pic': True}
+
+source_urls = ["https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs919/"]
```

Comment:
@wpoely86 don't hardcode the version in here please, use `%(version_major)s%(version_minor)s`